prov/bgq/configure.m4

```diff
@@ -0,0 +1,91 @@
+dnl Configury specific to the libfabrics BGQ provider
+
+dnl Called to configure this provider
+dnl
+dnl Arguments:
+dnl
+dnl $1: action if configured successfully
+dnl $2: action if not configured successfully
+dnl
+AC_DEFUN([FI_BGQ_CONFIGURE],[
+	# Determine if we can support the bgq provider
+	bgq_happy=0
+	bgq_direct=0
+
+	bgq_driver=/bgsys/drivers/ppcfloor
+	AC_SUBST(bgq_driver)
+	AC_ARG_WITH([bgq-driver],
+		[AS_HELP_STRING([--with-bgq-driver=@<:@BGQ driver installation path@:>@],
+			[Provide path to where BGQ system headers are installed])
+		],
+		[bgq_driver=$with_bgq_driver])
+
+	bgq_driver_CPPFLAGS="-I$bgq_driver -I$bgq_driver/spi/include/kernel/cnk"
```

Comment:
You should probably verify that these directories exist.  It's much more user-friendly to fail during `configure` if someone fat-fingers `--with-bqg-driver=/oops/i/typed/it/wrong` vs. failing during `make` (remember: with automake silent rules, people will just see that the compile failed due to a missing header file -- they won't see the exact compilation line unless they `make V=1`).